test/test_verifyLeaf.js

```diff
@@ -0,0 +1,65 @@
+const { expect } = require("chai")
```

Comment:
This file looks broken – is it?

Reply:
Yeah looks like it, the hardhat TypeScript types seem to be broken. I didn't set them up and in the interest of getting MMR verification done, I wrote the tests for this task in Truffle to avoid the time sync of fixing the hardhat tests up. Created issue https://github.com/Snowfork/verification-solidity/issues/9 to address the hardhat tests.